source/install/config-proxy-apache2.rst

```diff
@@ -49,11 +45,11 @@ Copy the `default` configuration file found in the same directory.
 
 		</VirtualHost>
 
-5. Because you'll likely have not set up the subdomain before now on Apache2, run ``a2ensite mysubdomain.mydomain.com`` to enable the site (do not run ``a2ensite mysubdomain.mydomain.com.conf``).
+5. (Debian/Ubuntu only:) Because you'll likely have not set up the subdomain before now on Apache2, run ``a2ensite mysubdomain.mydomain.com`` to enable the site (do not run ``a2ensite mysubdomain.mydomain.com.conf``).
 
-6. Restart Apache2
+6. Restart Apache2.
 
 	- On Ubuntu 14.04 and RHEL 6: ``sudo service apache2 restart``
-	- On Ubuntu 16.04 and RHEL 7: ``sudo systemctl restart apache2``
+	- On Ubuntu 16.04+ and RHEL 7+: ``sudo systemctl restart apache2``
 
-You should be all set! Ensure that your Mattermost config file is pointing to the correct URL and then ensure that once deployed your socket connection is not dropping.
+You should be all set! Ensure that your Mattermost config file is pointing to the correct URL (which may include a port), and then ensure that once deployed your socket connection is not dropping. To prevent external access to Mattermost on port 8065, in the config file, set "ListenAddress" to "localhost:8065" instead of ":8065".
```

Comment:
```suggestion
You should be all set! Ensure that your Mattermost config file is pointing to the correct URL (which may include a port), and then ensure that your socket connection is not dropping once deployed. To prevent external access to Mattermost on port 8065, in the config file, set ``ListenAddress`` to ``localhost:8065`` instead of ``:8065``.
```